### PR TITLE
Enabled disabled translation

### PR DESCRIPTION
**Pull request by mkz212**
Zrzut ekranu 2023-11-7 o 14 42 51

### CODE REVIEW

**Comment by bwp91** [comment]
FYI - in the ui repo, if you `ts-node scripts/lang-sync.ts` this will populate the other language files

**Comment by mkz212** [comment]
I'm happy to do it, just could you tell me how? I'm not very proficient in github.

**Comment by mkz212** [comment]
I added it manually. But I only translated in pl .

File: ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html [hunk 1]
```diff
@@ -83,6 +83,20 @@
           <th scope="row" class="text-nowrap" [translate]="'status.widget.systeminfo.label_plugin_path'">Plugin Path</th>
           <td>{{ serverInfo.homebridgeCustomPluginPath }}</td>
         </tr>
+        <tr *ngIf="serverInfo.homebridgeRunningInDocker">
```
Comment (bwp91) [inline ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html:78]: My thoughts on the docker info row is just to show it if the user is running homebridge in docker.

I don't think we need to show a 'No' if the user is not.

File: ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html [hunk 2]
```diff
@@ -83,6 +83,20 @@
...
+          <th scope="row" class="text-nowrap" [translate]="'status.widget.systeminfo.label_docker'">Docker</th>
+          <td class="text-nowrap" [translate]="'status.widget.systeminfo.label_yes'">Yes</td>
+        </tr>
+        <tr *ngIf="serverInfo.homebridgeRunningInSynologyPackage">
```
Comment (bwp91) [inline ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html:82]: I don't think we need to show this row at all. The OS shows as 'Synology DSM' (line 28) if the user is on this.

File: ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html [hunk 3]
```diff
@@ -83,6 +83,20 @@
...
+          <th scope="row" class="text-nowrap" [translate]="'status.widget.systeminfo.label_synology_package'">
+            Synology Package
+          </th>
+          <td class="text-nowrap" [translate]="'status.widget.systeminfo.label_yes'">Yes</td>
+        </tr>
+        <tr *ngIf="serverInfo.homebridgeServiceMode">
```
Comment (bwp91) [inline ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html:92]: I like the idea of showing this service row, but at the top just under OS.

I think we could make use of the 'arrow-up' icon like on line 46, and if the user is not running in service mode, then show a tooltip to say that is recommended that they do, linking to this wiki page: https://homebridge.io/w/JUvQr

File: ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html (hunk 1 above)
Comment (bwp91) [reply ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html:78]: this row already exists on line 31

File: ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html (hunk 3 above)
Comment (bwp91) [reply ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html:92]: and service mode row is already on line 35

**Comment by mkz212** [comment]
Something must have been very confused because I only added the translation of the word Yes. I'm about to see through it.

**Comment by mkz212** [comment]
Ok, corrected.

**Review by donavanbecker — APPROVED** [review]

**Review by donavanbecker — APPROVED** [review]

**Review by donavanbecker — APPROVED** [review]

File: ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html (hunk 3 above)
Comment (donavanbecker) [reply ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html:92]: Resolving as I only see `<tr *ngIf="serverInfo.homebridgeServiceMode">` on line 35 now.

File: ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html (hunk 2 above)
Comment (donavanbecker) [reply ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html:82]: looks like this was resolved.

File: ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html (hunk 1 above)
Comment (donavanbecker) [reply ui/src/app/modules/status/widgets/system-info-widget/system-info-widget.component.html:78]: only seeing this `<tr *ngIf="serverInfo.homebridgeRunningInDocker">` on line 31. Resolving

**Pull request merged**